.hermione.conf.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  sets: {
+    desktop: {
+      files: 'test/regression'
+    }
+  },
+
+  baseUrl: 'http://localhost:8000',
+  gridUrl: 'http://localhost:4444/wd/hub',
+  compositeImage: true,
+
+  browsers: {
+    chromeXL: {
+      desiredCapabilities: {
+        browserName: 'chrome',
+        chromeOptions: {
+          args: ['--headless'],
+        },
+      },
+      windowSize: '1300x900',
+    },
```

Comment:
```suggestion
    },
    chromeXS: {
      desiredCapabilities: {
        browserName: 'chrome',
        chromeOptions: {
          args: ['--headless'],
        },
      },
      windowSize: '500x300',
    },
```

Reply:
will it work?

Reply:
Nope, the window size is too small so some elements required to conduct test will not be visible